config/secrets.yml

```diff
@@ -12,7 +12,7 @@
 
 development:
   secret_key_base: 3270e3c985ea0649533e1745a2ed124ef28d475b694152d6f3165ae9507463bfd5ea3bc15b048b2e3a8530c3a0496e85856bb616ae74d1a821402b209f94e538
```

Comment:
Are we using these secret keys for anything? If not, lets remove them. If we are using them and its uniquely assigned by a third-party API service, lets not track this file.